build_container/build_container_ubuntu.sh

```diff
@@ -12,6 +12,10 @@ apt-get install -y --no-install-recommends locales software-properties-common ap
 # set locale
 localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
 
+# gdb
+add-apt-repository -y ppa:ubuntu-toolchain-r/test
+apt-get update -y
```

Comment:
probably don't need this here since we get it below as well, might save a bit of time